### ISSUE TITLE
Timeline interaction events firing

### ISSUE THREAD

**Issue by aMUSiC** [swimlane/ngx-charts#1511]
**Is your feature request related to a problem? Please describe.**
When I am displaying a big set of data within a chart (let's say 16.000 nodes), the sheer amount of data causes the rendering to be sluggish. To overcome this, I'm trying to thin the nodes on the server side (larger granularity for bigger date ranges) and repopulate the graph data whenever I zoom into a section using the timeline. However, the timeline selection does not fire any events so I cannot get a precise range programmatically to fire a subsequent data request to the server.

**Describe the solution you'd like**
I'd like to be able to capture timeline related events (select/drag) which return a range of the selected nodes (either first/last in the range, or the entire range) so I can use the information for subsequent handling

**Comment by jziggas** [comment]
Yes please, great idea! We are working with several thousand entries and noticed sluggish behavior when using the timeline as well. It would be really amazing to be able to only load the data that is needed based on the date range that a user selects.
